site/tsconfig.json

```diff
@@ -23,8 +23,8 @@
       "@components/*": ["components/*"],
       "@commerce": ["../packages/commerce/src"],
       "@commerce/*": ["../packages/commerce/src/*"],
-      "@framework": ["../packages/local/src"],
-      "@framework/*": ["../packages/local/src/*"]
+      "@framework": ["../packages/bigcommerce/src"],
+      "@framework/*": ["../packages/bigcommerce/src/*"]
```

Comment:
@okbel Let's make sure this isn't changed when merging 🙏

Reply:
```suggestion
      "@framework": ["../packages/local/src"],
      "@framework/*": ["../packages/local/src/*"]
```